expat/doc/xmlwf.xml

```diff
@@ -379,11 +394,19 @@ supports both.
   <refsect1>
   <title>OUTPUT</title>
     <para>
-	If an input file is not well-formed,
+	If any input file is not well-formed,
 	<command>&dhpackage;</command> prints a single line describing
-	the problem to standard output.  If a file is well formed,
-	<command>&dhpackage;</command> outputs nothing.
-	</para>
+	the problem to standard output. If the <option>-k</option> option is
+	provided, <command>&dhpackage;</command> continues processing the
+	remaining input files, describing problems found in any of them.
+	If <option>-k</option> is not provided, <command>&dhpackage;</command>
+	exits after processing the first input file with a problem.
+	<option>-k</option> outputs nothing for files which are well-formed.
+    </para>
+    <para>
+        <command>&dhpackage;</command> exits with code 0 if all its input
+        is well-formed and with code 2 if any input fails to be well-formed.
```

Comment:
There are also exit statuses 1 and 3 and the man page already has a section on that. My vote for dropping that paragraph.